### PR TITLE
Convert Heroes to atomic structure

### PR DESCRIPTION
**Pull request by Scotchester**
This PR will convert Heroes to the new atomic structure.

## Additions

- A new `molecules/heroes.less` file within `src/cf-layout/src`

## Changes

- `.hero` ➡️  `.m-hero`

## Testing

- See instructions in #326 

## Review

- @jimmynotjim 
- @KimberlyMunoz 

## Todos

- Add `npm run link` support after #326 is merged.

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-layout/src/molecules/heroes.less [hunk 1]
```diff
@@ -0,0 +1,128 @@
+@hero-desktop-height: 285px;
```
Comment (jimmynotjim) [inline src/cf-layout/src/molecules/heroes.less]: Left align these values to match the rest of the project.

File: src/cf-layout/src/molecules/heroes.less [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+@hero-image-height: 195px;
+@hero-bg: #f7f8f9;
```
Comment (jimmynotjim) [inline src/cf-layout/src/molecules/heroes.less]: Add corresponding 18F standards variables in a comment to match the rest of the project.

File: src/cf-layout/src/molecules/heroes.less [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+        .respond-to-min( @bp-sm-min, {
+            padding-right: unit( 15px / @base-font-size-px, em );
+            padding-left: unit( 15px / @base-font-size-px, em );
+        } );
+
+        .respond-to-min( @bp-med-min, {
+            padding-top: unit( 45px / @base-font-size-px, em );
+            padding-bottom: unit( 45px / @base-font-size-px, em );
+        });
```
Comment (jimmynotjim) [inline src/cf-layout/src/molecules/heroes.less]: Minor, add spaces inside parens

File: src/cf-layout/src/molecules/heroes.less [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        .respond-to-max( @bp-sm-max, {
+            .heading-2();
+        } );
+    }
+
+    &_subhead,
+    &_cta:not( .btn ) {
+        .webfont-regular();
+
+        font-size: @size-iv;
```
Comment (jimmynotjim) [inline src/cf-layout/src/molecules/heroes.less]: Should we add a comment why we aren't using the `.heading-#` vars here?

File: src/cf-layout/src/molecules/heroes.less [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+        .respond-to-min( @bp-sm-min, {
+            .m-hero_image {
+                // Overwrite the image but don't remove the container from
+                // the grid flow. Necessary due to table-cell.
+                background: transparent !important;
+            }
+        } );
+    }
+
+    &__white-text {
```
Comment (jimmynotjim) [inline src/cf-layout/src/molecules/heroes.less]: Should we change this to `__overlay-text` in this update?

**Comment by Scotchester** [comment]
Pushed an update to address @jimmynotjim's comments.

**Comment by jimmynotjim** [comment]
👍

**Comment by jimmynotjim** [comment]
You'll need to rebase the upstream `prerelease-ver-4` for this to work in the sandbox.

**Comment by Scotchester** [comment]
@jimmynotjim Rebased.

**Pull request merged**